recipes/livvkit/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "livvkit" %}
+{% set version = "2.1.5" %}
+{% set nbuild = 0 %}
+{% set sha256 = "f6e016b1b95942189159845ba4cfa19c0147a394d33e8633216c684af64dfc94" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }} 
+
+build:
+  noarch: python
+  number: {{ nbuild }}
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+  - livv = livvkit.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - numpy
```

Comment:
Can you open an issue upstream about removing numpy from setup_requires?

Reply:
I've opened LIVVkit/LIVVkit#41